posturemanagementv1/posture_management_v1_integration_test.go

```diff
@@ -0,0 +1,179 @@
+				Skip("External configuration file not found, skipping tests: " + err.Error())
+			}
+
+			os.Setenv("IBM_CREDENTIALS_FILE", externalConfigFile)
+			config, err = core.GetServiceProperties(posturemanagementv1.DefaultServiceName)
+			if err != nil {
+				Skip("Error loading service properties, skipping tests: " + err.Error())
+			}
+			serviceURL = config["URL"]
+			if serviceURL == "" {
+				Skip("Unable to load service URL configuration property, skipping tests")
+			}
+
+			accountID = config["ACCOUNT_ID"]
+			Expect(accountID).ToNot(BeEmpty())
+
+			profileName = config["PROFILE_NAME"]
+			Expect(profileName).ToNot(BeEmpty())
+
+			scopesName = config["SCOPES_NAME"]
+			Expect(scopesName).ToNot(BeEmpty())
+
+			fmt.Printf("Service URL: %s\n", serviceURL)
+			shouldSkipTest = func() {}
+		})
+	})
+
+	Describe(`Client initialization`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It("Successfully construct the service client instance", func() {
+
+			postureManagementServiceOptions := &posturemanagementv1.PostureManagementV1Options{}
+
+			postureManagementService, err = posturemanagementv1.NewPostureManagementV1UsingExternalConfig(postureManagementServiceOptions)
+
+			Expect(err).To(BeNil())
+			Expect(postureManagementService).ToNot(BeNil())
+			Expect(postureManagementService.Service.Options.URL).To(Equal(serviceURL))
+		})
+	})
+
+	Describe(`ListProfiles - List profiles`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`ListProfiles(listProfilesOptions *ListProfilesOptions)`, func() {
+
+			listProfilesOptions := &posturemanagementv1.ListProfilesOptions{
+				AccountID: core.StringPtr(accountID),
+				Name:      core.StringPtr(profileName),
+			}
+
+			profilesList, response, err := postureManagementService.ListProfiles(listProfilesOptions)
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(profilesList).ToNot(BeNil())
+
+			profileID = profilesList.Profiles[0].ProfileID
+		})
+	})
+
+	Describe(`ListScopes - List scopes`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`ListScopes(listScopesOptions *ListScopesOptions)`, func() {
+
+			listScopesOptions := &posturemanagementv1.ListScopesOptions{
+				AccountID: core.StringPtr(accountID),
+				Name:      core.StringPtr(scopesName),
+			}
+
+			scopesList, response, err := postureManagementService.ListScopes(listScopesOptions)
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(scopesList).ToNot(BeNil())
+
+			scopeID = scopesList.Scopes[0].ScopeID
+		})
+	})
+
+	Describe(`CreateValidation - Initiate a validation`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`CreateValidation(createValidationOptions *CreateValidationOptions)`, func() {
+			Expect(scopeID).ToNot(BeNil())
+			Expect(profileID).ToNot(BeNil())
+
+			createValidationOptions := &posturemanagementv1.CreateValidationOptions{
+				AccountID:      core.StringPtr(accountID),
```

Comment:
When dealing with a variable like `accountID`, you can avoid using the core function and just specify `&accountID` instead.   The StringPtr() functions are mainly for when we're dealing with literals.